spec/services/collect_stats_spec.rb

```diff
@@ -8,7 +8,7 @@
   it "collects the stats from icecast's xml" do
     radio = FactoryBot.create :radio, name: "datafruits"
     VCR.use_cassette(RSpec.current_example.metadata[:full_description].to_s) do
-      CollectStats.new(radio).perform
+      CollectStats.new(radio).perform 
```

Comment:
I decided to mark this spec as pending https://github.com/datafruits/streampusher-api/pull/84
Can you pull and merge master ?

Reply:
Done